cmd/oras/root/discover.go

```diff
@@ -101,8 +102,9 @@ Example - Discover referrers of the manifest tagged 'v1' in an OCI image layout
 	}
 
 	cmd.Flags().StringVarP(&opts.artifactType, "artifact-type", "", "", "artifact type")
-	cmd.Flags().StringVarP(&opts.Format.FormatFlag, "output", "o", "tree", "[Deprecated] format in which to display referrers (table, json, or tree). tree format will also show indirect referrers")
+	cmd.Flags().StringVarP(&opts.Format.FormatFlag, "output", "o", "tree", "[Deprecated] format in which to display referrers (table, json, or tree).")
 	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", false, "display full metadata of referrers")
+	cmd.Flags().IntVarP(&opts.depth, "depth", "", 20, "level of indirect referrers to display")
```

Comment:
Any ideas about a better default value than 20?

Reply:
If the specified depth is less or equal than 0, it should be considered as infinity. And we can use `0` as the default value.

Reply:
If we use 0 to represent infinity, we will need to do extra explanation to the users and it may cause confusion.

Reply:
Updated design: default is set to 0. If user does not give a value for `--depth`, we show all levels of referrers (use the `isChanged` function of flags). If user enters a value smaller than 1, we return an error.